README.md

```diff
@@ -1,3 +1,48 @@
 # qe-metrics
 
-Report QE metrics
+Queries Jira and stores results in a database for further analysis in Grafana.
+
```

Comment:
please add db schemas +  explanation; i'm ok with doing it in a separate PR. in this case, please add `TODO`